src/services/api/role.ts

```diff
@@ -57,6 +58,6 @@ export class RoleService {
 	}
 
 	public getDefaultRole() {
-		return this.roleRepository.findOneBy({ name: 'default' });
```

Comment:
I suspect this is the reason why your tests are failing. LogTo role `default` is a special type of role AFAIK and it was used for unauthenticated user tests, i.e., any APIs users could use without being logged in at all. (We did have such APIs initially, e.g., DID search, DID get which was basically DID resolution.) I don't think we have this any more.

Reply:
Huh, I checked the value of the const and not what I was expecting it to be. 🤔

Reply:
This one is only fetching it from the database. We do store logto role ids in DB too and this one is a helper method to find the default role